modules/core/src/smithy4s/capability/MonadThrowLike.scala

```diff
@@ -18,7 +18,8 @@ package smithy4s.capability
 
 import smithy4s.kinds.PolyFunction
 
-trait MonadThrowLike[F[_]] extends Zipper[F] {
```

Comment:
@daddykotex, this is a bit annoying for downstream, because it means that there are few more constructs to implement to use the generic routers, but it feels more "right", as otherwise there are some annoying bits wrt orphan instances, which make the decoders less usable.

Reply:
Decided to move instances to orphan locations instead. It's more pragmatic.